scripts/cococo/evaluations_movable_logical_qubits/fix_q_vary_depth/vary_depths_hex_251103.py

```
@@ -0,0 +1,317 @@
+    gates_list,
+    depths_mean_log,
+    yerr=depths_std_log,
+    fmt=".--",
+    color="yellowgreen",
+    capsize=3,
+    label="Depth Log.",
+)
+plt.xlabel("Num. Gates / Log. Depth")
+plt.ylabel("Depth of Schedule")
+plt.xscale("log")
+plt.grid(True, which="both", ls="--", alpha=0.7)
```

Ruff (0.14.11)

Boolean positional value in function call

(FBT003)